scripts/protogen.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 # Generates protobuf Go datastructures from the proto directory.
 
-SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
-PROTOBUF_DIR=${PROTOBUF_DIR-${SCRIPT_DIR}/proto}
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)/.."
```

Comment:
This should be `+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}/..")" && pwd -P)"`